cmd/bucket-lifecycle.go

```diff
@@ -157,11 +157,13 @@ func initBackgroundExpiry(ctx context.Context, objectAPI ObjectLayer) {
 	ewk, err := workers.New(workerSize)
 	if err != nil {
 		logger.LogIf(ctx, err)
+		return
```

Comment:
actually this PR is wrong we should always initialize ILM and never return errors, initialize this with some random valid value.

```
```suggestion
		ewk = 4
```